src/nvapi/nvapi_d3d_low_latency_device.cpp

```diff
@@ -42,6 +42,15 @@ namespace dxvk {
         return m_applicationIdList[((lowLatencyDeviceFrameId - 1) % applicationIdListSize)];
     }
 
+    bool LowLatencyFrameIdGenerator::IsRepeatedFrame(uint64_t frameID, uint32_t markerType) {
+        if (m_lastFrameId.contains(markerType) && m_lastFrameId[markerType] == frameID) {
+            return true;
+        }
+
+        m_lastFrameId[markerType] = frameID;
+        return false;
```

Comment:
```suggestion
        return std::exchange(m_lastFrameId[markerType], frameID) == frameID;
```

While this does have different behavior for `frameID == 0`, we can fix it by changing type of `m_lastFrameId` to `std::unordered_map<uint32_t, std::optional<uint64_t>>` (some of our tests rely on this).

But I still believe it should be an array instead of a map though :see_no_evil: